tests.py

```diff
@@ -100,6 +100,25 @@ def test_doc_path(self):
         text = fulltext.get('files/test.doc', backend='doc')
         self.assertEqual(text, TEXT_WITH_NEWLINES)
 
+OCR_TEXT_START = (
+	u"Sherlock Holmes and Doctor Watson lived at 2211) Baker Street between 1881-1904,\n"
+)
+OCR_TEXT_END = (
+	u"Step back in time, and when you visit London, remember to visit the world's most\n"
+	u"famous address!"
+)
+				
+@add_metaclass(FullTextFilesMeta)
+class FullTextFiles(unittest.TestCase):
```

Comment:
This class name is already declared (so this declaration overwrites it). There is no need to declare another class, simply add your two methods to the existing `FullTextFiles` class.